core/components/Notification.vue

```diff
@@ -16,27 +16,26 @@ export default {
   },
   created () {
     this.$bus.$on('notification', data => {
-      if (this.notifications.length > 0 && this.notifications[this.notifications.length - 1].message === data.message) {
```

Comment:
Why is that removed? It was to block the stacked set of the same messages which sometimes happen for example when user clicks 10times add to cart